.github/workflows/slack-release-notifier.yml

```diff
@@ -0,0 +1,15 @@
+name: "Slack release notifications"
+on:
+  release:
+    types: [published]
+jobs:
+  slack-release-notifier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - uses: ./tools/slack-release-notifier/
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_RELEASE_BOT_TEST }}
```

Comment:
@ffriedl89 Please update to send to the right channel

Reply:
I'd rather test it on one release first and change the URL afterwards :)